src/components/__tests__/SystemPreferences.spec.js

```diff
@@ -0,0 +1,45 @@
+import { mount } from '@vue/test-utils'
+import SystemPreferences from '../SystemPreferences.vue'
+const deepmerge = require('deepmerge');
+
+function createWrappedPage(props) {
+  return mount(SystemPreferences, { propsData: props });
+}
+
+const baseProps = {
+  memoryInGB: 4,
+  numberCPUs: 5,
+  availMemoryInGB: 8,
+  availNumCPUs: 6
+};
+
+describe('SystemPreferences.vue', () => {
```

Comment:
Do we need some tests for actual UI manipulation? Is that even reasonable?

Reply:
I assume that pulling in all those graphic libraries to run jest mean we can do some kind of UI testing. I haven't looked at it yet.

Reply:
It's definitely a longer-term goal -- right now we need to get something for the eng summit.